src/SAWScript/Interpreter.hs

```diff
@@ -4068,7 +4068,7 @@ primitives = Map.fromList
     "HeapsterEnv -> String -> String -> String -> String -> TopLevel HeapsterEnv"
     (bicVal heapster_assume_fun_rename)
     Experimental
-    [ "heapster_assume_fun_rename nm nm_t perms trans assumes that function nm"
+    [ "heapster_assume_fun_rename nm nm_to perms trans assumes that function nm"
```

Comment:
Now that I look closer, I think this prose is missing the name of the `HeapsterEnv` argument, which should come before `nm`. Similarly for `heapster_assume_fun`.